README.md

```diff
@@ -177,6 +177,7 @@ A selection of websites offering high quality video stock.
 completely free.
 * [Videezy](https://www.videezy.com) - [:copyright:](https://www.videezy.com/terms) Free stock videos, 4k footage, and free HD video clips.
 * [Vidsplay](http://www.vidsplay.com/) - [:copyright:](http://www.vidsplay.com/terms.html) Free HD stock footage (Attribution required).
+* [VYOO](http://www.veed.io/vyoo) - Free Vertical Videos by [VEED.IO](http://www.veed.io/).
```

Comment:
Add a copyright link to the license in which the videos are provided and make a not of the license in the description.